great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -543,13 +543,25 @@ def _build_s3_object_key(self, key):
         return s3_object_key
 
     def _get(self, key):
+        client = self._create_client()
         s3_object_key = self._build_s3_object_key(key)
+        return self._get_by_s3_object_key(client, s3_object_key)
 
-        s3 = self._create_client()
+    @override
+    def _get_all(self) -> list[Any]:
+        """Get all objects from the store.
+        NOTE: This is non-performant because we download each object separately.
```

Comment:
Is there any reference to docs/code we can put here?

Reply:
good call. Updated.